server/events/approve_policies_command_runner_test.go

```diff
@@ -57,10 +66,12 @@ func TestApproveCommandRunner_IsOwner(t *testing.T) {
 					PolicySets: valid.PolicySets{
 						Owners: valid.PolicyOwners{
 							Users: c.OwnerUsers,
+							Teams: c.OwnerTeams,
 						},
 					},
 				},
 			}, nil)
+			When(vcsClient.GetTeamNamesForUser(fixtures.GithubRepo, fixtures.User)).ThenReturn(c.OwnerTeams, nil)
```

Comment:
Is it better to add another test case that user team is different with with policy set owner team?

Reply:
Yeah that's fair, added a test for that.